TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOpConfig.java

```diff
@@ -3,10 +3,53 @@
 
 @Config
 public class TeleOpConfig {
-    public static double CLAW_LEFT_CLOSED = 0;
```

Comment:
What do the numbers next to the equal sign indicate?

Reply:
Assalamu Alaikom Ali. No need to @ arshad specifically. He might have exams at this time and someone else can answer inshAllah.

Good question. So, this line is to assign a variable.
- CLAW_LEFT_CLOSED: The variable name 
- The value being assigned to it is 0.
- The variable is of type "double" (decimal basically)
- and it is public (meaning can be accessed from outside the class (TeleOpConfig))
- and it is static which means you can do something like TeleOpConfig. CLAW_LEFT_CLOSED anywhere in the code and it will work.

Resources to help you learn java in general and ask even better questions: https://books.trinket.io/thinkjava2/
Reach out on discord if you have questions on this: https://books.trinket.io/thinkjava2/